src/main/kotlin/com/modules/db/reposInterfaces/BasicInterface.kt

```diff
@@ -0,0 +1,14 @@
+package com.modules.db.reposInterfaces
+
+import org.jetbrains.exposed.dao.IntEntity
+import org.jetbrains.exposed.dao.IntEntityClass
+import org.jetbrains.exposed.dao.id.EntityID
+import org.jetbrains.exposed.dao.id.IntIdTable
+
+
+interface BasicInterface <ModelT> {
```

Comment:
`BasicInterface` to zbyt generyczna nazwa żeby cokolwiek znaczyła, nie rozumiem też jaka za nią stoi idea